frontend/pages/admin/OrgSettingsPage/cards/Sso/Sso.tsx

```diff
@@ -181,19 +181,23 @@ const Sso = ({
             parseTarget
             onBlur={validateForm}
```

Comment:
TODO: I noticed that the `InputField` component doesn't pass the `onBlur` prop down if the type is `textarea`. Do we want to change that? What might break if we do?